### PR TITLE
Replaced focus modal with popup just above timeline

### LINKED ISSUE #6706: Focus Window modal 🔫
https://www.figma.com/file/mAotNZRv6M5X0zHIFKPzFY/Replay-Design-Doc?node-id=12025%3A76141
https://www.loom.com/share/33bbeae6bdcf4e448a1ce33e384a7f81

### PR DESCRIPTION
**Pull request by bvaughn**
Resolves #6706

Loom walk through: https://www.loom.com/share/980a68226f9846a6bb8c8ae647a8c51e

# Screenshots
## Before
Replay focus mode (dark theme) before changes
Replay focus mode (light theme) before changes

## After
Replay focus mode (dark theme) after changes
Replay focus mode (light theme) after changes

### CODE REVIEW

File: src/ui/components/Timeline/FocusModePopout.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      >
+        <div className="flex h-6 w-6 items-center justify-center rounded-full bg-primaryAccent text-white">
+          <Icon filename="focus" className="bg-iconColor" />
+        </div>
+
+        <div className="grow text-sm">
+          <strong>Focus mode</strong> lets you debug a subset of the replay.{" "}
+          <a
+            href="https://docs.replay.io/docs/viewer-26591deb256c473a946d0f64abb67859#bf19baaa57004b0d9282cc0a02b281f5"
+            rel="noreferrer"
```
[the remark this answers is not in the document]
Comment (bvaughn) [reply src/ui/components/Timeline/FocusModePopout.tsx]: Would be nice if our linter caught this too. It catches "noreferrer" but not "noopener".

**Review by jaril — APPROVED** [review]
Played around with it and works well. LGTM!

**Comment by bvaughn** [comment]
> Something like "You changed some values, and we're about to forget about it. You sure about this?". But that might be a follow-up that's already been discussed, and I don't feel too strongly about it.

Yeah let's tackle this in follow up if others have the same concern. :)

I don't think we use the "prompt before discarding" pattern anywhere else (like pending comments) although _maybe we should_ 😁

**Comment by jonbell-lot23** [comment]
To discuss later: https://www.notion.so/replayio/When-to-delete-user-input-8bea4556c48743b985f55c8c129f2897#b65df61c49854ee9a8c8960867c91729

Cc @bvaughn @jaril

**Pull request merged**